Dockerfile

```diff
@@ -14,6 +14,7 @@
 # NOTE: since updateBaseImages.sh does not support other registries than RHCC, update to RHEL8
 # https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/devtools/go-toolset-rhel7
 ARG DEV_WORKSPACE_CONTROLLER_VERSION="master"
+ARG DEV_WORKSPACE_CHE_OPERATOR_VERSION="main"
```

Comment:
This makes me wonder. Is `ARG DEV_WORKSPACE_CONTROLLER_VERSION="master"` correct, @tolusha ? Because https://github.com/devfile/devworkspace-operator switched to using `main` as well...